rust/src/cli_experimental.rs

```diff
@@ -31,12 +31,17 @@ enum ComposeCmd {
         #[clap(flatten)]
         opts: crate::compose::BuildChunkedOCI,
     },
+    TreeToRootfs {
```

Comment:
Bikeshed: could just call this `Rootfs`. We're already under `compose` and `compose rootfs` flows better. (Or I guess if we want to be verbose, `compose rootfs-from-treefile`?)